source/crud/delete.txt

```diff
@@ -51,6 +51,14 @@ To delete all documents from a collection, pass an empty
    more information and examples, see
    :method:`~db.collection.deleteMany()`.
 
+.. note::
+
+   If you want to delete all documents from a large collection, dropping
+   and recreating the collection may have faster performance than
+   deleting documents with the :method:`db.collection.deleteMany()`
+   method. When you recreate the collection, you must also recreate any
+   specified collection characteristics such as collection indexes.
```

Comment:
The word "characteristics" stand out to me. Is this a word that's used elsewhere?

Reply:
I changed to "parameters" to be more specific!